runtime.txt

```diff
@@ -1 +1 @@
-python-2.7.14
+python-2.7.15
```

Comment:
Should also update the installation docs

Reply:
Wait no, that's for CCI itself, never mind

Reply:
It's the version for Heroku to run in its dyno :)